src/utils.ts

```diff
@@ -1,25 +1,26 @@
-/* @flow */
-/* eslint max-lines: 0 */
-
 import { isRegex, noop } from "./util";
-import type {
-  CrossDomainWindowType,
-  SameDomainWindowType,
-  DomainMatcher,
+import {
```

Comment:
Nit: should we use this syntax instead since they are all types?
```ts
import type { CrossDomainWindowType, SameDomainWindowType, DomainMatcher } from "./types";
```

Reply:
I slightly prefer the syntax Greg is suggesting.  If we wanted to, I think we could enforce this with an eslint rule: 

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md#fixstyle